janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java

```diff
@@ -604,6 +604,15 @@ public boolean apply(@Nullable String s) {
             "Size of the batch in which mutations are persisted",
             ConfigOption.Type.MASKABLE, 1024, ConfigOption.positiveInt());
 
+    /**
+     * Number of mutations in a transaction after which use parallel processing for transaction aggregations.
+     * This might give a boost in transaction commit time.
+     * Default value is 100.
+     */
+    public static final ConfigOption<Integer> NUM_MUTATIONS_PARALLEL_THRESHOLD = new ConfigOption<>(STORAGE_NS,"num-mutations-parallel-threshold",
+        "Number of mutations after which use parallel processing for transaction aggregations.",
+    ConfigOption.Type.MASKABLE, 100, ConfigOption.positiveInt());
+
```

Comment:
I think it also makes sense to improve the parameter description, so that new users understand the reason behind this parameter:
```
This parameter determines the minimum number of mutations a transaction must have before parallel processing is applied during aggregation. Leveraging parallel processing can enhance the commit times for transactions involving a large number of mutations. However, it is advisable not to set the threshold too low (e.g., 0 or 1) due to the overhead associated with parallelism synchronization. This overhead is more efficiently offset in the context of larger transactions.
```

When you introduce a new configuration parameter in JanusGraph, it's necessary to update the documentation to include this new parameter. This can be achieved by initiating the process to rebuild the configuration parameters documentation. This process automatically integrates the newly added parameter into the JanusGraph documentation. To proceed, follow the steps below:
```
mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am
git add -A
git commit --amend -s
git push -f
```
Anytime you build JanusGraph (with `mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am` or `mvn --quiet clean install -DskipTests=true`) it will update configuration reference documentation if there are any changes in the doc.
See more here: https://github.com/JanusGraph/janusgraph/blob/master/BUILDING.md#building-documentation

Reply:
Nice! I was wondering myself what is the process to update it.